packages/edit-site/src/hooks/commands/use-edit-mode-commands.js

```diff
@@ -72,7 +90,7 @@ function usePageContentFocusCommands() {
 			),
 			icon: layout,
 			callback: ( { close } ) => {
-				setRenderingMode( 'template-only' );
+				editTemplate.onClick();
```

Comment:
With the removal of `template-mode` we are now using the `getPostLinkProps.onClick` in a few places that are not actually click events so I am thinking we should rename it to something like `editPost` ? What do you think @talldan ?

Reply:
🤦 Doh! Fixed.